Makefile

```diff
@@ -0,0 +1,65 @@
+DOCKER_COMPOSE ?= docker compose
+FILE ?= local.yml
+DC = $(DOCKER_COMPOSE) --file=$(FILE)
+RUN = $(DC) run --rm
+DJANGO = django
+MANAGE = $(RUN) $(DJANGO) python manage.py
+
+shell:
+	$(MANAGE) shell
+
+
+logs:
+	$(DC) logs -f $(DJANGO)
+
+
+down:
+	$(DC) down
+
+
+up:
+	$(DC) up -d
+
+
+restart: down up
+
+
+build:
+	$(DC) build
+
+
+createsuperuser:
+	$(MANAGE) createsuperuser
+
+
+freshstart: down build up
```

Comment:
I might suggest renaming this to `rebuild` instead of `freshstart` - I would expect the `freshstart` command to delete the postgres docker volume as well

Reply:
Actually, probably we should have a `freshstart` as well, so what's the best way to delete the postgres volumes (aside from `volume prune` 😂)

Reply:
https://stackoverflow.com/questions/45511956/remove-a-named-volume-with-docker-compose

😱

Reply:
YOOOOOOOOO